File: logic-exercises/low-level/exe11.c
#include <stdio.h>

int main(void) {
    int n, a[3];
    
    printf("Digite um valor: ");
    scanf("%d", &n);

    for (int i = 1; i <= n; i++) {
        a[0] = i;
        a[1] = i * i;
        a[2] = i * i * i;
        printf("%d %d %d\n", a[0], a[1], a[2]);
    }
}

File: logic-exercises/eclipse/exe11.java
package eclipse;

import java.util.Scanner;

public class exe11 {
    static public void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite um valor: ");
        int n = scanner.nextInt();

        for (int i = 1; i <= n; i++) {
            int a = i;
            int b = i * i;
            int c = i * i * i;
            System.out.println(a + " " + b + " " + c);
        }
        scanner.close();
    }
}